lms/data_tasks/report/create_from_scratch/99_grant_permissions/01_grant_schema.jinja2.sql

```diff
@@ -1,11 +1,6 @@
 -- The public schema already exists but we need to grant usage to the user we
 -- will map via FDW.
 
-{% for fdw_user in fdw_users %}
-    -- Once this has been run once, it can be removed
-    REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA public FROM "{{fdw_user}}";
-{% endfor %}
```

Comment:
I'm not sure if this would happen naturally, and we could skip a PR, but hey ho. Best to be careful